### DISCUSSION TITLE
Yank text into register and then play it with "@"

### DISCUSSION THREAD

**Discussion by whatwouldmarvindo** [VSCodeVim/Vim#7326] [Q&A]
I found this tutorial on YouTube that goes a little in depth about registers in vim and how to execute registers.

I tried to replicate the vim behavior by doing the following:
1. recording a makro with `q<register>`
2. replaying the makro with `@<register>`
3. pasting the content of the register with `"<register>p`
4. editing the pasted makro and save it into a register with `0"<register>y$ `

Replaying the edited register with `@<register>` doesn't work now. Is this a bug or is there anything I am missing?

Thank you in advance :)

**Comment by elazarcoh** [comment]
It's an open issues. See #1658.
I might try to implement it (I already made a proof of concept of it), but the solution is not very elegant, so no promises it would get merged any time soon.
